shared/models/plugins/server-hook.model.ts

```diff
@@ -9,9 +9,13 @@ export const serverFilterHookObject = {
   // Used to get detailed video information (video watch page for example)
   'filter:api.video.get.result': true,
 
-  // Filter the result of the accept upload function
+  // Filter the result of the accept upload, import via torrent or url functions
   // If this function returns false then the upload is aborted with an error
   'filter:api.video.upload.accept.result': true,
+  'filter:api.video.pre-import-url.accept.result': true,
```

Comment:
Please specify the difference between a `pre-import` and a `post-import`

Reply:
:+1: